packages/cos/build.yaml

```diff
@@ -108,3 +108,9 @@ excludes:
 - ^/usr/local/share
 - ^/usr/local/src
 - ^/usr/local/games
+
+- ^/etc/hostname
```

Comment:
I've added those to excludes. Hostname is generated by docker, so not really needed, similarly for the rest.